stock_tracked_fifo_valuation/models/stock_move.py

```diff
@@ -11,36 +11,45 @@ class StockMove(models.Model):
     @api.model
     def _get_in_base_domain(self, company_id=False):
         domain = [
-            ('state', '=', 'done'),
-            ('location_id.usage', '!=', 'internal'),
-            ('location_dest_id.company_id', '=', company_id or self.env.user.company_id.id),
-            ('location_dest_id.usage', '=', 'internal'),
+            ("state", "=", "done"),
```

Comment:
Note to Check: Is this even needed any more since upstream changed these functions?